### ISSUE TITLE
Cannot automatically switch back to the root folder

### ISSUE THREAD

**Issue by fermion-boson** [Macro-Deck-App/Macro-Deck#482]
**Describe the bug**
When I add a new auto-switch folder, I open the corresponding program and switch to the corresponding folder, but when I exit the program, it does not switch back to the configuration of the root folder

**Comment by Elleby77** [comment]
I don't think this would be considered a bug,  but its certainly a good feature to have.    I've been trying to figure out how to do this and there doesn't seem to be a method for it.

**Comment by hellishvictor** [comment]
> **Describe the bug** When I add a new auto-switch folder, I open the corresponding program and switch to the corresponding folder, but when I exit the program, it does not switch back to the configuration of the root folder

Same here using v2.13.1; I just need that it switch folders/profiles depending on active window, as Stream Deck does.

**Comment by manuelmayer-dev** [comment]
This issue is related to Macro Deck 2, which is no longer actively developed.

Macro Deck 3 is now the active version. If this issue is still relevant in Macro Deck 3, please create a new issue in the Macro Deck 3 repository so it can be tracked separately.
